### ISSUE TITLE
Using English for code changes

### ISSUE THREAD

**Issue by codefx9** [go-xorm/xorm#380]
Hi,
Thanks for this great library. But sometimes it is very difficult to follow changes made to this lirary, when all the communication in a pull request or issue is in Chinese. Will it be possible always summarize code changes in English for a _merged pull request_ where all or most prior messages are in Chinese? 

Chrome does not suggest auto translate for Github. As a non-chinese speaker, it is hard to follow those cases. If I could go one step further, may I suggest that titles of issues and pull requests are encouraged to be in English (may be mentioned in the readme file), so that at least non-chinese speakers like me know when to dig in further into an issue or pull request with the help of Google translate.

Thanks.

**Comment by lunny** [comment]
Hi, I think this is a good idea. I will add some a language description on README for issues and PRs. Thanks!

**Comment by lunny** [comment]
I have added a notice on `CONTRIBUTING.md`.
